bi_sql_editor/models/bi_sql_view.py

```diff
@@ -293,7 +293,8 @@ def unlink(self):
                     "If you want to delete them, first set them to draft."
                 )
             )
-        self.cron_id.unlink()
+        if self.cron_id:
+            self.cron_id.unlink()
```

Comment:
More simple : you could write  in one line.
Self.mapped("cron_id").unlink()

Reply:
@legalsylvain Thanks for the review! I updated the code to use mapped.

I tested with only the self.mapped("cron_id").unlink() without the if, and it raises the same error, for that reason I leave the if statement.

Reply:
> without the if, and it raises the same error

owh. That's weird. unlinking a recordset of 0 element should not raise an error.

never mind. Thanks for the patch.